source/common/upstream/ring_hash_lb.h

```diff
@@ -65,6 +66,9 @@ class RingHashLoadBalancer : public ThreadAwareLoadBalancerBase,
 
     std::vector<RingEntry> ring_;
 
+    std::vector<uint64_t> ring_shard_;
+    uint64_t rightShift = 9; // SHARD_SHIFT parameter
```

Comment:
1. might need snake_case here to satisfy the format check.
2. Can we use a better variable name here?

Reply:
I tried to make the change @suniltheta. Please let me know if ok or suggest further change. Thanks